<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QWidget" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>main_window</string>
  </property>
  <property name="styleSheet">
   <string notr="true"> #main_window{
border-radius: 10px;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="b_Head_portrait">
         <property name="minimumSize">
          <size>
           <width>59</width>
           <height>59</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>59</width>
           <height>59</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border:0px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Resources/1.png</normaloff>Resources/1.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="b_1">
         <property name="maximumSize">
          <size>
           <width>59</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>第一页</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="b_2">
         <property name="minimumSize">
          <size>
           <width>59</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>59</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>第二页</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>59</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="b_3">
         <property name="minimumSize">
          <size>
           <width>59</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>59</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border:0px;
font: 75 10pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>教务系统</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Resources/学校.png</normaloff>Resources/学校.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>38</width>
           <height>38</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton">
         <property name="minimumSize">
          <size>
           <width>59</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>59</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
border:0px;

</string>
         </property>
         <property name="text">
          <string>实践中心</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Resources/实践教育管理.png</normaloff>Resources/实践教育管理.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>38</width>
           <height>38</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(245, 245, 245);</string>
       </property>
       <property name="currentIndex">
        <number>4</number>
       </property>
       <widget class="QWidget" name="page">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QWebEngineView" name="webEngineView_0">
           <property name="url">
            <url>
             <string>about:blank</string>
            </url>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>340</x>
           <y>230</y>
           <width>221</width>
           <height>131</height>
          </rect>
         </property>
         <property name="text">
          <string>第二页</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_3"/>
       <widget class="QWidget" name="page_4">
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>710</x>
           <y>40</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_5"/>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
